rancher2/resource_rancher2_custom_user_token.go

```diff
@@ -10,14 +10,25 @@ import (
 	managementClient "github.com/rancher/rancher/pkg/client/generated/management/v3"
 )
 
-func resourceRancher2ServiceAccountToken() *schema.Resource {
+/*
+The Rancher API can only create tokens for the user owning the token used to authenticate the API call. To acquire
+the initial token to connect to the Rancher API a username+password login is needed. The lifespan of this token is
+controlled by Rancher.
+
+The following steps are used to create a Terraform controlled Rancher API token:
+1) Acquire a temporarily Rancher API token for the custom user by doing a username+password login
+2) Create a Terraform controlled token by authenticating using the (temporarily) token from step 1
+3) Logout the temporarilu token acquired in step 1
```

Comment:
Nit `Logout using the temporarily acquired token in step 1`

Reply:
opened #1130 for it